### PR TITLE
fix: Make sure CDK deployments use our CDK bootstrap stack

### PR DESCRIPTION
**Pull request by mnapoli**

### CODE REVIEW

File: components/website/src/WebsiteConstruct.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default class WebsiteConstruct extends Stack {
   private readonly props: WebsiteInput;
 
   constructor(scope: Construct, id: string, props: WebsiteInput) {
-    super(scope, id);
+    super(scope, id, {});
```
Comment (mnapoli) [inline components/website/src/WebsiteConstruct.ts:24]: TODO: fix

File: core-aws/src/Cdk.ts [hunk 2]
```diff
@@ -37,16 +40,9 @@ export default class Cdk {
...
-       * The `:` is a shell "no-op" command. The `--app` arg tells CDK which command to run
-       * to package the application. Since we package the app ourselves above (via `synth()`),
-       * the artifacts exist already. The CDK can directly take these artifacts and deploy.
-       */
+      // Skip synthesis since we package the app ourselves above (via `synth()`)
+      // The CDK can directly take these artifacts and deploy.
       '--app',
-      ':',
-      '--toolkit-stack-name',
-      this.toolkitStackName,
```
Comment (mnapoli) [inline core-aws/src/Cdk.ts:48]: Just confirming as well that `--toolkit-stack-name` had indeed no effect (it was for v1 bootstrap). Removing 🧹

Comment (pgrzesik) [reply core-aws/src/Cdk.ts:48]: great call 👍

**Review by pgrzesik — APPROVED** [review]
Looking great 👍

**Pull request merged**